src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const stringPeople = document.querySelectorAll('.population');
+const numberPeople = [];
+
+for (const ch of stringPeople) {
```

Comment:
What is `ch`? Use meaningful variable names, always. The name of the variables should make it clear what is happening in the code